elfi/utils.py

```diff
@@ -96,3 +96,71 @@ def get_sub_seed(random_state, sub_seed_index, high=2**31):
         n_unique = len(seen)
 
     return sub_seeds[-1]
+
+
+def tabulate(funs, *args):
+    """Compute a function on the cartesian product of the arguments.
```

Comment:
I don't see the need for duplicating the functionality of `np.vectorize` and `np.meshgrid`.